curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f44f10ea40f300b896a5e.md

```diff
@@ -19,6 +19,18 @@ You should not have an `if` statement.
 assert.notMatch(code, /if\s*\(/);
 ```
 
+You should not have an `else if` statement.
```

Comment:
```suggestion
You should not use an `else if` statement.
```

Reply:
It looks like other assert statements use similar language when asking campers to remove code.

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/07-scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/65687c2fd0fec7ba9fb8af30.md

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657f28a0482132aca51a9212.md

I vote we keep this as is for consistency 👍